src/manifest/Application.ts

```diff
@@ -62,28 +63,27 @@ export class Application {
   public store: ManifestStore;
 
   /**
-   * An optional property defining if the application is created without using the extensibility SDK
-   * by manually parsing of the iframe URL containing the context value parameters.
+   * An optional property defining if the application is created without using the extensibility SDK.
+   * This apps are manually parsing of the iframe URL containing the context value parameters and not
+   * sending the READY message back to the Outreach host so the timeout can not be monitored due to that.
    *
    * If omitted in manifest, the default value is false.
    *
+   * @see https://github.com/getoutreach/extensibility-sdk/blob/main/docs/url-parsing.md#timeout-handling
    * @type {boolean}
    * @memberof Application
-   * @deprecated use disableTimeoutMonitoring instead
    */
-
-  public notUsingSdk?: boolean;
+  public disableTimeoutMonitoring?: boolean;
 
   /**
-   * An optional property defining if the application is created without using the extensibility SDK.
-   * This apps are manually parsing of the iframe URL containing the context value parameters and not
-   * sending the READY message back to the Outreach host so the timeout can not be monitored due to that.
+   * An optional section containing the webhook definition enabling Outreach
+   * to send appropriate event information over webhook
    *
-   * If omitted in manifest, the default value is false.
+   * @see https://github.com/getoutreach/extensibility-sdk/blob/master/docs/manifest.md#webhooks
+   * @see https://github.com/getoutreach/extensibility-sdk/blob/master/docs/webhooks.md
    *
-   * @see https://github.com/getoutreach/extensibility-sdk/blob/main/docs/url-parsing.md#timeout-handling
-   * @type {boolean}
+   * @type {ManifestWebhook}
    * @memberof Application
    */
-  public disableTimeoutMonitoring?: boolean;
+  public webhooks?: ManifestWebhook;
```

Comment:
The plural name applies that we support multiple webhooks, but it is actually a single webhook with a single URL.

Did you consider using singular `webhook` name here or allowing multiple webhooks (thus allowing calling multiple URLs) ?

Reply:
Asking Anton about this and I'll rename if it is only one

Reply:
renamed